proposed_changes.txt

```diff
@@ -0,0 +1,81 @@
+Some ideas on modifying data representation.
+This commit is not intended to be merged. It is only intended for discussion.
+If it is decided that some changes make sense, then they shall be handled by multiple separate commits or pull requests.
+
+Idea 1:
+
+Use float (or in rare cases double) instead of int-based datatypes for all datapoints where fractional numbers might be relevant.
+(for datapoints like Axle.WheelCount it is natural to use an int-based type)
+
+Idea 2: 
+
+Stick to the same representation for all datapoints using a particular unit, e.g. always use float for km/h.
```

Comment:
Would be solved by making all floats/numbers.

I just always felt SI Units would be better (so only m/s). But that is another topic